### ISSUE TITLE
TestStartStop: "Failed to setup certs" flake

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#7541]
as seen here 
https://storage.googleapis.com/minikube-builds/logs/7435/70adc95/Docker_Linux.html#fail_TestStartStop%2fgroup%2fold-docker

```
	W0408 20:09:24.196997    4909 exit.go:101] Failed to setup certs: certificate symlinks: create symlink for /usr/share/ca-certificates/4970.pem: sudo /bin/bash -c "test -s /usr/share/ca-certificates/4970.pem && ln -fs /usr/share/ca-certificates/4970.pem /etc/ssl/certs/4970.pem": exit status 1
	stdout:
	
	stderr:
	* 
	X Failed to setup certs: certificate symlinks: create symlink for /usr/share/ca-certificates/4970.pem: sudo /bin/bash -c "test -s /usr/share/ca-certificates/4970.pem && ln -fs /usr/share/ca-certificates/4970.pem /etc/ssl/certs/4970.pem": exit status 1
	stdout:
	
	stderr:
	
	* 
	* minikube is exiting due to an error. If the above message is not useful, open an issue:
```

**Comment by medyagh** [comment]
CC: @tstromberg

**Comment by tstromberg** [comment]
```
	I0408 20:09:15.512190    4909 certs.go:342] found cert: /home/jenkins/minikube-integration/linux-amd64-docker-7435-2451-70adc95be27abfd1f53736cd23f13206bf501118/.minikube/certs/home/jenkins/minikube-integration/linux-amd64-docker-7435-2451-70adc95be27abfd1f53736cd23f13206bf501118/.minikube/certs/4970.pem (1338 bytes)
	W0408 20:09:15.512248    4909 certs.go:338] ignoring /home/jenkins/minikube-integration/linux-amd64-docker-7435-2451-70adc95be27abfd1f53736cd23f13206bf501118/.minikube/certs/home/jenkins/minikube-integration/linux-amd64-docker-7435-2451-70adc95be27abfd1f53736cd23f13206bf501118/.minikube/certs/4970_empty.pem, impossibly tiny 0 bytes
...
	I0408 20:09:22.527884    4909 certs.go:120] copying: /usr/share/ca-certificates/4970.pem
	I0408 20:09:23.815705    4909 kic_runner.go:91] Run: openssl version
	I0408 20:09:23.983664    4909 kic_runner.go:91] Run: sudo /bin/bash -c "test -s /usr/share/ca-certificates/4970.pem && ln -fs /usr/share/ca-certificates/4970.pem /etc/ssl/certs/4970.pem"
...
	I0408 20:09:23.983664    4909 kic_runner.go:91] Run: sudo /bin/bash -c "test -s /usr/share/ca-certificates/4970.pem && ln -fs /usr/share/ca-certificates/4970.pem /etc/ssl/certs/4970.pem"

	W0408 20:09:24.196997    4909 exit.go:101] Failed to setup certs: certificate symlinks: create symlink for /usr/share/ca-certificates/4970.pem: sudo /bin/bash -c "test -s /usr/share/ca-certificates/4970.pem && ln -fs /usr/share/ca-certificates/4970.pem /etc/ssl/certs/4970.pem": exit status 1
```

**Comment by tstromberg** [comment]
Great news! We now have a more interesting error message:

`W0410 04:25:13.826351   26334 out.go:201] Failed to setup certs: Copy /home/jenkins/minikube-integration/linux-amd64-docker-7580-2643-10f924df4a5d1a7a5312424fb220970b28adf485/.minikube/certs/5144.pem: write: /tmp/tmpf-memory-asset852732570: expected to write 0 bytes, but wrote 1338 instead`

https://storage.googleapis.com/minikube-builds/logs/7580/10f924d/Docker_Linux.html#fail_TestStartStop%2fgroup%2fembed-certs

**Comment by priyawadhwa** [comment]
Can be closed via #7723
